### PR TITLE
Update local-api page

### PR DESCRIPTION
**Pull request by efb4f5ff-1298-471a-8973-3d47447115dc**
link fixes

### CODE REVIEW

**Review by PikachuEXE — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]

File: usage/local-api.md [hunk 1]
```diff
@@ -26,18 +26,17 @@ The Local API is one method of obtaining data from YouTube. For another method o
 
 ## Modules Used in the Local API
 
-| Name  | License | Functionality | Maintained By the FreeTube Team?
-| ------------- | ------------- | ------------- |
-| [youtube-chat](https://github.com/LinaTsukusu/youtube-chat)  | MIT | Live Chat | No |
-| [youtube-suggest](https://github.com/goto-bus-stop/youtube-suggest)  | Apache-2.0 | Search Suggestions | No |
-| [ytdl-core](https://github.com/fent/node-ytdl-core)  | MIT  | Obtain Video Information | No |
-| [ytpl](https://github.com/TimeForANinja/node-ytpl)  | MIT  | Playlists | No |
-| [node-ytsr](https://github.com/TimeForANinja/node-ytsr)  | MIT  | Search Functionality | No |
```
Comment (absidue) [inline usage/local-api.md:35]: Why was ytsr removed?

Comment (absidue) [reply usage/local-api.md:35]: https://github.com/FreeTubeApp/FreeTube/blob/development/package.json#L84

Comment (efb4f5ff-1298-471a-8973-3d47447115dc) [reply usage/local-api.md:35]: i must be blind, will add back!

**Review by absidue — APPROVED** [review]

**Review by ChunkyProgrammer — APPROVED** [review]
did not notice the ytsr removal

**Review by PikachuEXE — APPROVED** [review]

**Pull request merged**